dnf5/context.cpp

```diff
@@ -799,7 +803,7 @@ bool Context::check_gpg_signatures(const libdnf::base::Transaction & transaction
 void Context::download_and_run(libdnf::base::Transaction & transaction) {
     download_packages(transaction, nullptr);
 
-    std::cout << std::endl << "Verifying GPG signatures" << std::endl;
+    std::cout << std::endl << "Verifying PGP signatures" << std::endl;
```

Comment:
When changing all output from GPG -> PGP, maybe also the line 748 here in the `context.cpp`?